mono/metadata/object-internals.h

```diff
@@ -2160,6 +2160,8 @@ mono_string_empty_handle (MonoDomain *domain);
 gpointer
 mono_object_get_data (MonoObject *o);
 
+#define mono_get_data_unsafe(handle) ((guint8*)MONO_HANDLE_RAW (handle) + MONO_ABI_SIZEOF (MonoObject))
```

Comment:
Why the need to diverge so much from the existing `mono_object_get_data`? Neither the return type, nor the name is equivalent. How about you try to make it more consistent?

Reply:
Return type good point, I raised that, but missed the consistency point, and is already fixed, except for the exact name of the same type, I can fix that too.

On the name, well, change "object" to "handle"? I considered that, but there was the prior goal of not adding "handle" to names as well, because these are the new better names and shouldn't be warty. Perhaps we have done that enough though -- lots of functions with "handle" in name, and saying nothing is too vague, even with "object" or "handle" it seems vague, and this is now worse, so I'll do that.

Ultimately with C++ the name can be identical via overloading and that would be ideal presumably in terms of name choice, assuming the original name is good, and reducing churn, but we aren't here.